week4:JSweek2/5.js

```diff
@@ -0,0 +1,5 @@
+var z = '7.25';
```

Comment:
this should be initialised as a number, not a string ;)

Reply:
got rid of ' ' :)